student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission, name)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] As requested by the previous reviewer please make sure the `parent_permission` parameter has a default value as `true`.